File-Formats/Presentation/FAQ.md

```diff
@@ -48,6 +48,18 @@ Current version of the Essential Presentation library does not provide support f
 
 You can improve the quality of converted images by specifying the image resolution. Refer – [Converting PowerPoint presentation to Images](/file-formats/presentation/getting-started#converting-powerpoint-presentation-to-images)
 
+## Why images are preserved as RedX images in PPTX to PDF/Image conversion?
```

Comment:
Why metafile (*.wmf, *.emf) and TIFF images are not preserved in PPTX to PDF/Image conversion?

Presentation doesn't preserves those during the PPTX to PDF or Image conversions to avoid pagination problems.